serde_codegen/src/ser.rs

```diff
@@ -478,7 +516,7 @@ fn serialize_tuple_variant(
     structure_ty: P<ast::Ty>,
     fields: &[ast::StructField],
     field_names: Vec<Ident>,
```

Comment:
Unrelated to this PR, but why do we have field names for tuple fields? We forward to `SeqVisitor` anyway afaik... is this leftover from an early serde?